sunsynk/definitions.py

```diff
@@ -146,18 +145,67 @@
 ###########
 _SENSORS += (
     Sensor(200, "Control Mode"),
-    Sensor(201, "Equalization voltage", VOLT, 0.01),
-    Sensor(202, "Absorption voltage", VOLT, 0.01),
     Sensor(230, "Grid Charge Battery current", AMPS, -1),
     Sensor(232, "Grid Charge enabled", "", -1),
     Sensor(312, "Battery charging voltage", VOLT, 0.01),
     Sensor(603, "Bat1 SOC", "%"),
     Sensor(611, "Bat1 Cycle"),
 )
-BATTERY_SHUTDOWN_CAPACITY = RWSensor(217, "Battery Shutdown Capacity", "%")
-BATTERY_RESTART_CAPACITY = RWSensor(218, "Battery Restart Capacity", "%")
-BATTERY_LOW_CAPACITY = RWSensor(219, "Battery Low Capacity", "%")
-_SENSORS += (BATTERY_SHUTDOWN_CAPACITY, BATTERY_RESTART_CAPACITY, BATTERY_LOW_CAPACITY)
+
+# Absolute min and max voltage based on Deye inverter
+MIN_VOLTAGE = 41
+MAX_VOLTAGE = 60
+
+EQUALIZATION_VOLTAGE = NumberRWSensor(
+    201, "Equalization voltage", VOLT, 0.01, min=MIN_VOLTAGE, max=MAX_VOLTAGE
+)
+ABSORPTION_VOLTAGE = NumberRWSensor(
+    202, "Absorption voltage", VOLT, 0.01, min=MIN_VOLTAGE, max=MAX_VOLTAGE
+)
+FLOAT_VOLTAGE = NumberRWSensor(
+    203, "Float voltage", VOLT, 0.01, min=MIN_VOLTAGE, max=MAX_VOLTAGE
+)
+
+BATTERY_SHUTDOWN_CAPACITY = NumberRWSensor(217, "Battery Shutdown Capacity", "%")
+BATTERY_RESTART_CAPACITY = NumberRWSensor(218, "Battery Restart Capacity", "%")
+BATTERY_LOW_CAPACITY = NumberRWSensor(
+    219,
+    "Battery Low Capacity",
+    "%",
+    min=BATTERY_SHUTDOWN_CAPACITY,
+    max=BATTERY_RESTART_CAPACITY,
+)
```

Comment:
These sensors are currently called `Battery Shutdown Capacity`, `Battery Restart Capacity` and `Battery Low Capacity`. Similarly there are new `Battery Shutdown voltage`, `Battery Restart voltage` and `Battery Low voltage` sensors. I am wondering whether the `Battery` prefix should be removed since the Equalization, Absorption and Float voltage sensors do not have this prefix and perhaps there should be consistency. It would also read better here if the prefixes were gone, but would it be obvious that these sensors are battery-related?
<img width="293" alt="image" src="https://user-images.githubusercontent.com/16413523/187090468-7f1ec59a-aa2d-433d-a529-ed2b7464c236.png">

Reply:
I would rather add "Battery" to absorption and float than remove it from the rest.

Reply:
Done. I also added deprecated entries for `Equalization voltage` and `Absorption voltage` since they were present prior to the beta.